lib/kitchen/driver/openstack.rb

```diff
@@ -380,8 +387,8 @@ def disable_ssl_validation
         Excon.defaults[:ssl_verify_peer] = false
       end
 
-      def wait_for_server(state)
-        state[:hostname] = get_ip(state)
+      def wait_for_server(server, state)
+        state[:hostname] = get_ip(server)
```

Comment:
This should probably be in #create_server instead of here.  This isn't really an action that is required during the wait_for_server activity.  Methods should be single-purpose :smile: